testsuite/test_wbemconnection_mock.py

```diff
@@ -2161,12 +2270,15 @@ class TestInstanceOperations(object):
 
     @pytest.mark.parametrize(
         "ns, cln, inst_id, tst_ns, exp_er", [
-            [None, 'CIM_Foo', 'CIM_Foo1', None, None],
-            ['root/blah', 'CIM_Foo', 'CIM_Foo1', 'root/blah', None],
-            ['blah', 'CIM_Foo', 'badid', 'blah', CIM_ERR_NOT_FOUND],
-            ['blah', 'CIM_Foo', 'CIM_Foo1', 'whoop',
+            [DEFAULT_NAMESPACE, 'CIM_Foo', 'CIM_Foo1', None, None],
```

Comment:
Original test was None for ns for this test.

Reply:
Same response as above.